app/src/main/java/org/wikipedia/dataclient/mwapi/TemplateDataResponse.kt

```diff
@@ -56,6 +62,6 @@ class TemplateDataResponse : MwResponse() {
 
         private val deprecatedAsString get() = deprecated?.jsonPrimitive?.contentOrNull
 
-        val isDeprecated get() = !deprecatedAsString.equals("false", true)
+        val isDeprecated get() = deprecatedAsString.equals("true", true)
```

Comment:
<img src="https://github.com/wikimedia/apps-android-wikipedia/assets/2435576/63aa6482-5db9-4116-8741-d1bc0cfbf039" width="480" />

<img src="https://github.com/wikimedia/apps-android-wikipedia/assets/2435576/4284dcf1-06a3-490a-a7b7-539a382c97d6" width="480"/>

The default value of `deprecated` is `false`, but it will become a `String` if it is deprecated.

Reply:
ahh of course. Should be fixed now.